packages/bbui/src/Table/Table.svelte

```diff
@@ -272,6 +272,7 @@
             {#if sortedRows?.length && fields.length}
               {#each sortedRows as row, idx}
                 <tr
+                  on:click={() => dispatch("click", row)}
```

Comment:
Didn't actually know you can do multiple `on:click` handlers with svelte like that, awesome.

Reply:
Yeah, definitely some nice syntactic sugar!